### DISCUSSION TITLE
part-ii/2019-6/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#56] [Question comments]
Part II 2019 Algebraic Topology: Paper 2, Section II, F 

https://questions.tripos.org/part-ii/2019-6/

**Comment by artie2000** [comment]
I get $\pi_1(S-K)=\langle a,b\mid a^2b^{-3}\rangle$.

**Reply by eha1234** [reply]
I agree
